stories/header-date-selector.stories.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * Header with Date Selector stories.
```

Comment:
I don't think separate stories are needed for this, let's combine them with `stories/header.stories.js` and we can add a decorator for receiving common setup into the registry instead. Look for usages of `addDecorator` in stories for an example 👍

Reply:
Combined using `addDecorator`.